<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JapaneseDictForm</class>
 <widget class="QWidget" name="JapaneseDictForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Japanese dictionary</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit1">
       <property name="font">
        <font>
         <family>Bitstream Cyberbit</family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditK">
       <property name="font">
        <font>
         <family>Bitstream Cyberbit</family>
         <pointsize>24</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>clear</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="font">
          <font>
           <family>Bitstream Cyberbit</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="font">
        <font>
         <family>Bitstream Cyberbit</family>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BStdout">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;save the highlighted line, unlike pressing Return on the upper line which save the first line&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>save entry</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BSong">
       <property name="toolTip">
        <string>Ctrl-F</string>
       </property>
       <property name="text">
        <string>Grep Lyrics</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+F</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit_alt1">
       <property name="font">
        <font>
         <family>AoyagiKouzanFont2</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>This is intended to display the text with alternative fonts or fontsize
You can edit it either with Qt Designer or with the .ui xml form (look for &quot;lineEdit_alt&quot;)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_alt2">
       <property name="font">
        <font>
         <family>UmePlus P Gothic</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>lineEdit1</tabstop>
  <tabstop>pushButton2</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>BStdout</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>766</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit_alt1</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>lineEdit_2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lineEdit_alt2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
